08_multibundle_requirejs/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "rendr-example-multibundle-requirejs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha --ui bdd --reporter spec  ./test --recursive",
+    "start": "node index.js",
+    "postinstall": "test -f ../../package.json && npm install ../../ || echo",
+    "postupdate": "test -f ../../package.json && npm install ../../ || echo"
+  },
+  "dependencies": {
+    "amdefine": "~0.1.0",
```

Comment:
`global-define` has this as it's dependency, so you probably don't need it at this level.